src/ui/components/SecondaryToolbox/react-devtools/injectReactDevtoolsBackend.ts

```diff
@@ -246,21 +271,10 @@ export const nodesToFiberIdsCache: Cache<
   async load([replayClient, pauseId, store]) {
```

Comment:
We should be able to remove the `store` parameter from this cache now.

lol we can probably delete that comment on line 269 as well